src/components/CustomerSelect.vue

```diff
@@ -80,18 +80,36 @@ export default defineComponent({
         showOnFocus: true,
         debounceWaitMs: 100,
         fetch: async (text, callback) => {
-          const {
-            data: { data: customers },
-          } = await new CustomersService().suggest(text);
+          let loadedData;
+          if (text === "") {
```

Comment:
Kannst du dieses if/else so refactoren, dass wir den Teil ab `const newData: ...` nur einmal brauchen. Der unterscheidet sich ja in den beiden blöcken nicht voneinander.